.env.docker

```
@@ -17,8 +17,8 @@ DC_DEMO_NEXT_PUBLIC_MEDUSA_BACKEND_URL=http://localhost:9000
 DC_DEMO_NEXT_PUBLIC_MEDUSA_PUBLISHABLE_KEY=
 
 #Medusa BE container
-DC_VITE_HMR_PORT=9009
-DC_DATABASE_URL=postgresql://root:root@medusa-db:5432/medusa?ssl_mode=disable
+DC_DATABASE_URL=postgresql://root:root@medusa-db:5432/medusa?sslmode=disable
+DC_LEGACY_DATABASE_URL=mysql://root:1234@engine-db:3306/n1shop
 DC_MINIO_FILE_URL="http://medusa-minio:9004/medusa-bucket"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)